dm/dm-precheck.md

```diff
@@ -111,6 +111,25 @@ For the incremental data migration mode (`task-mode: incremental`), in addition
 
 For the full and incremental data migration mode (`task-mode: all`), in addition to the [common check items](#common-check-items), the precheck also includes the [full data migration check items](#check-items-for-full-data-migration) and the [incremental data migration check items](#check-items-for-incremental-data-migration).
 
+### Ignorable check items
+
+Prechecks can find potential risks in your environments. It is not recommended to ignore check items. If your data migration task has special needs, you can use the [`ignore-checking-items` configuration item](/dm/task-configuration-file-full.md#task-configuration-file-template-advanced) to skip some check items.
+
+| Check item  | Description   |
+| :---------- | :------------ |
+| `dump_privilege`         | Checks the dump privilege of the user in the upstream MySQL instance. |
```

Comment:
this description contains "user", but it is disappeared in the "replication_privilege"